app/views/claims/_one_time_password.html.erb

```diff
@@ -27,9 +27,9 @@
 
       <div class="govuk-!-margin-bottom-6">
         <% if email_or_mobile == "email" %>
-          <%= govuk_link_to "Resend passcode (you will be sent back to the email address page)", claim_path(current_journey_routing_name, "email-address"), no_visited_state: true %>
+          <%= govuk_link_to "Resend passcode (you will be sent back to the email address page)", claim_path(current_journey_routing_name, "email-address", resend: true), no_visited_state: true %>
```

Comment:
```suggestion
          <%= govuk_link_to "Resend passcode (you will be sent back to the email address page)", claim_path(current_journey_routing_name, "email-address", claim: { resend: true }), no_visited_state: true %>
```

Does setting the resend params like this work and let us avoid having to set the `@resend` instance variable? If so we can lose the callbacks for setting `@resend` and not need to use `@resend` in the view.

Reply:
Using `params[:resent]` in the view avoids the need for the callbacks. 
I couldn't get `claim: { resend: true }` to work here - but I'll try a couple of other things before giving up.